charms/istio-ingressgateway/requirements.txt

```diff
@@ -0,0 +1,4 @@
+ops
```

Comment:
We should probably have version specifiers on these, if nothing other than `<x.0`, in case any of them publish breaking changes.